ods_ci/tests/Tests/0100__platform/0101__deploy/0101__installation/0101__post_install.robot

```diff
@@ -400,22 +400,26 @@ Verify DSC Contains Correct Component Versions  # robocop: disable:too-long-test
     ${component_versions_json} =    Evaluate     json.loads("""${component_versions}""")    json
     ${components} =  List Directories In Directory    ${RHODS_OPERATOR_GIT_DIR}/prefetched-manifests
     FOR  ${c}  IN  @{components}
-        IF    "${c}" == "aipipelines"
-            # The name in the prefetched-manifests has been kept as datasciencepipelines, so need this workaround
-            # to make the test work
-            ${c} =  Set Variable     "datasciencepipelines"
-        END
         ${component_metadata_file} =  Set Variable
         ...     ${RHODS_OPERATOR_GIT_DIR}/prefetched-manifests/${c}/component_metadata.yaml
         ${file_exists} =  Run Keyword And Return Status    File Should Exist  ${component_metadata_file}
         IF  ${file_exists}
-            IF  $component_versions_json[$c] == {} or $component_versions_json[$c]["managementState"] != "Managed"
-                Log  ${c} is not managed, skipping version check
+            IF    "${c}" == "datasciencepipelines"
+                ${cmp} =  Set Variable   aipipelines
```

Comment:
@CFSNM  do we actually need a new variable here? Can we just reassign `${c}` or does robot framework not allow that?